dhall-toml/dhall-toml.cabal

```
@@ -0,0 +1,78 @@
+Name:               dhall-toml
+Version:            0.1.0
+Cabal-Version:      >=1.10
+Build-Type:         Simple
+Tested-With:        GHC == 8.4.3, GHC == 8.6.1
+License:            BSD3
+License-File:       LICENSE
+Copyright:          2021 ear7h
+Author:             ear7h
+Maintainer:         Gabriel439@gmail.com
+Bug-Reports:        https://github.com/dhall-lang/dhall-haskell/issues
+Synopsis:           Convert between Dhall and TOML
+Description:
+    Use this package if you want to convert between dhall expressions and TOML.
+    You can use this package as a library or an executable:
+    .
+    * See "Dhall.TOML" or "Dhall.TOMLToDhall" modules if you want to use this
+      package as a library
+    * Use @dhall-to-toml@, @toml-to-dhall@ programs from this package if you
+      want an executable.
+    .
+    The "Dhall.TOML" and "Dhall.TOMLToDhall" modules also contains instructions
+    for how to use this package
+Category:           Compiler
+Extra-Source-Files:
+    README.md
+    CHANGELOG.md
+    tasty/data/*.dhall
+    tasty/data/*.toml
+
+Source-Repository head
+    Type: git
+    Location: https://github.com/dhall-lang/dhall-haskell/tree/master/dhall-toml
+
+Library
+    Hs-Source-Dirs:   src
+    Build-Depends:
+        base  >= 4.14   && < 4.15 ,
```

Since we have CI with GHC 8.6, supporting GHC 8.6 and up seems reasonable.

For the upper bound, I'd suggest using our current convention `< 5`. If a released version turns out to be incompatible, we can easily fix the bounds with a [Hackage revision](https://github.com/haskell-infra/hackage-trustees/blob/master/revisions-information.md).

https://gitlab.haskell.org/ghc/ghc/-/wikis/commentary/libraries/version-history is a good resource on the boot library versions.


```suggestion
        base  >= 4.12   && < 5 ,
```